test/functional/wallet_basic.py

```diff
@@ -587,7 +587,7 @@ def run_test(self):
 
         # Get all non-zero utxos together
         chain_addrs = [self.nodes[0].getnewaddress(), self.nodes[0].getnewaddress()]
-        singletxid = self.nodes[0].sendtoaddress(chain_addrs[0], self.nodes[0].getbalance(), "", "", True)
+        singletxid = self.nodes[0].sendall(recipients=[chain_addrs[0]])['txid']
```

Comment:
In 923d24583d826f4c6ecad30b185e0e043ea11dfc I think that this change can be removed as it is modified in the next commit anyways?